server/mscalendar/welcome_flow.go

```diff
@@ -84,6 +84,14 @@ func (wf *welcomeFlow) makeSteps() {
 		FalseButtonMessage:   "No - Do not notify me of new events",
```

Comment:
This may not be in the scope of this PR, but I was thinking we should use the language "when you are invited to an event", instead of "when you receive a new event", in the prompts above this line, as it is clearer what we mean by "event".

Also, above we use the language "while you are on a meeting", versus "while you are in a meeting". Just some small improvements for wordings. @larkox Thoughts on this?